kclvm/query/Cargo.toml

```diff
@@ -16,6 +16,7 @@ kclvm-ast-pretty = {path = "../ast_pretty"}
 kclvm-parser = {path = "../parser"}
 kclvm-sema = {path = "../sema"}
 kclvm-error = {path = "../error"}
+maplit = "1.0.2"
```

Comment:
Does this crate `maplit` have been used?

Reply:
fixed